cmd/otelcol/config/collector/config.d.linux/receivers/sqlserver.discovery.yaml

```diff
@@ -0,0 +1,106 @@
+# upgrades of the Collector.                                                        #
+#####################################################################################
+# sqlserver:
+#   enabled: true
+#   rule:
+#     docker_observer: type == "container" and any([name, image, command], {# matches "(?i)mssql"}) and not (command matches "splunk.discovery")
+#     host_observer: type == "hostport" and command matches "(?i)mssql" and not (command matches "splunk.discovery")
+#     k8s_observer: type == "port" and pod.name matches "(?i)mssql"
+#   config:
+#     default:
+#       username: splunk.discovery.default
+#       password: splunk.discovery.default
+#       server: '`host`'
+#       port: '`port`'
+#       resource_attributes:
+#         sqlserver.instance.name:
+#             enabled: true
+#       metrics:
+#         # Enable all OOTB dashboard default metrics
+#         sqlserver.batch.request.rate:
+#             enabled: true
+#         sqlserver.batch.sql_compilation.rate:
+#             enabled: true
+#         sqlserver.batch.sql_recompilation.rate:
+#             enabled: true
+#         sqlserver.database.count:
+#             enabled: true
+#         sqlserver.database.io:
+#             enabled: true
+#         sqlserver.database.latency:
+#             enabled: true
+#         sqlserver.database.operations:
+#             enabled: true
+#         sqlserver.lock.wait.rate:
+#             enabled: true
+#         sqlserver.page.buffer_cache.hit_ratio:
+#             enabled: true
+#         sqlserver.processes.blocked:
+#             enabled: true
+#         sqlserver.resource_pool.disk.throttled.read.rate:
+#             enabled: true
+#         sqlserver.resource_pool.disk.throttled.write.rate:
+#             enabled: true
+#         sqlserver.user.connection.count:
+#             enabled: true
+#         # Disable upstream default metrics that OOTB dashboards don't use
+#         sqlserver.lock.wait_time.avg:
+#             enabled: false
+#         sqlserver.page.checkpoint.flush.rate:
+#             enabled: false
+#         sqlserver.page.lazy_write.rate:
+#             enabled: false
+#         sqlserver.page.life_expectancy:
+#             enabled: false
+#         sqlserver.page.operation.rate:
+#             enabled: false
+#         sqlserver.page.split.rate:
+#             enabled: false
+#         sqlserver.transaction.rate:
+#             enabled: false
+#         sqlserver.transaction.write.rate:
+#             enabled: false
+#         sqlserver.transaction_log.flush.data.rate:
+#             enabled: false
+#         sqlserver.transaction_log.flush.rate:
+#             enabled: false
+#         sqlserver.transaction_log.flush.wait.rate:
+#             enabled: false
+#         sqlserver.transaction_log.growth.count:
+#             enabled: false
+#         sqlserver.transaction_log.shrink.count:
+#             enabled: false
+#         sqlserver.transaction_log.usage:
+#             enabled: false
+#
```

Comment:
```suggestion
# 
```
`make bundle.d` expects a whitespace here. We should fix it to not add an incorrect whitespace, but for now, we should just get the test to pass.